doc/jcli/vote.md

```diff
@@ -7,7 +7,45 @@ to decrypt and certify the tally.
 
 ## Creating committee keys
 
-TBA
+### Private
+Please refer to `jcli votes committee --help` for help with the committee related cli operations and specification of arguments.
+
+In this example we will be using 3 kind of keys for the private vote and tallying.
+
+In order:
+
+#### Committee communication key
+
+```shell
+jcli votes committee communication-key generate > ./comm.sk
+```
+
+We can get its public representation with:
+
+```shell
+jcli votes committee communication-key to-public --input ./comm.sk > ./comm.pk
+```
+
+#### Committee member key
+
+```shell
+crs=$(jcli vote crs generate)
```

Comment:
Is crs generated per committee member? AFAIK it should be the same for all members.

Reply:
It should be the same though.

Reply:
Please add a note that specifically says all members should use the same crs string

Reply:
typo - votes